tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -506,3 +513,71 @@ def mock_property(self):
         ):
             DelayedSaturatedMMM.load("test_model")
         os.remove("test_model")
+
+
+@pytest.mark.parametrize(
+    "dist, mu, observed, dims, expected_exception",
+    [
+        # This is a placeholder for the actual parameters you want to test with
+        # For example, a test case with an expected exception could be like:
+        (
+            {"dist": "NonExistentDistribution"},
+            np.array([0]),
+            np.array([10, 11, 9]),
+            "obs_dim",
+            ValueError,
+        ),
+        # Add more test cases as needed
+    ],
+)
+def test_create_likelihood_distribution(
+    mmm, dist, mu, observed, dims, expected_exception
+):
+    if expected_exception:
+        with pytest.raises(expected_exception):
+            mmm._create_likelihood_distribution(dist, mu, observed, dims)
+    else:
+        mmm._create_likelihood_distribution(dist, mu, observed, dims)
```

Comment:
We should test the actual thing we wanted was created when it is valid. There is no assertion in here. 

This line would check there is a `pm.Normal` in the graph: `assert any(isinstance(var.owner.op, pm.Normal) for node in pytensor.graph.ancestors(output) if var.owner)`